[DEBUG] [2020-10-29 08:12:03,558] [2468:140422157502272] - component_properties.py[line:67]: {'DataIOParam': {'input_format': 'dense', 'delimitor': ',', 'data_type': 'float64', 'exclusive_data_type': None, 'tag_with_value': False, 'tag_value_delimitor': ':', 'missing_fill': False, 'default_value': 0, 'missing_fill_method': None, 'missing_impute': None, 'outlier_replace': False, 'outlier_replace_method': None, 'outlier_impute': None, 'outlier_replace_value': 0, 'with_label': True, 'label_name': 'y', 'label_type': 'int', 'output_format': 'dense'}, 'initiator': {'role': 'guest', 'party_id': 10000}, 'job_parameters': {'work_mode': 0, 'model_id': 'arbiter-10000#guest-10000#host-10000#model', 'model_version': '202010290812019279649'}, 'role': {'guest': [10000], 'host': [10000], 'arbiter': [10000]}, 'config': '/fate/examples/federatedml-1.x-examples/homo_logistic_regression/test_homolr_evaluate_job_conf.json', 'dsl': 'examples/federatedml-1.x-examples/homo_logistic_regression/test_homolr_evaluate_job_dsl.json', 'function': 'submit_job', 'local': {'role': 'guest', 'party_id': 10000}, 'CodePath': 'federatedml/util/data_io.py/DataIO', 'module': 'DataIO'}
[DEBUG] [2020-10-29 08:12:03,559] [2468:140422157502272] - component_properties.py[line:74]: need_run: True, need_cv: False
[DEBUG] [2020-10-29 08:12:03,559] [2468:140422157502272] - component_properties.py[line:118]: has_train_data: False, has_eval_data: False, has_normal_data: True
[DEBUG] [2020-10-29 08:12:03,561] [2468:140422157502272] - model_base.py[line:210]: In extract_data, data input: {'args.data': <arch.api.impl.based_1x.table.DTable object at 0x7fb17aa83fd0>}
[DEBUG] [2020-10-29 08:12:03,561] [2468:140422157502272] - model_base.py[line:87]: func: <bound method DataIO.fit of <federatedml.util.data_io.DataIO object at 0x7fb17aa83668>>
[INFO] [2020-10-29 08:12:03,561] [2468:140422157502272] - data_io.py[line:141]: start to read dense data and change data to instance
[DEBUG] [2020-10-29 08:12:03,563] [2468:140422157502272] - data_io.py[line:95]: header is y,x0,x1,x2,x3,x4,x5,x6,x7,x8,x9,x10,x11,x12,x13,x14,x15,x16,x17,x18,x19,x20,x21,x22,x23,x24,x25,x26,x27,x28,x29
[DEBUG] [2020-10-29 08:12:03,563] [2468:140422157502272] - data_io.py[line:96]: sid_name is id
[DEBUG] [2020-10-29 08:12:03,645] [2468:140422157502272] - model_base.py[line:99]: saved_result is : [<arch.api.impl.based_1x.table.DTable object at 0x7f4d73068550>], data_output: <arch.api.impl.based_1x.table.DTable object at 0x7f4d73068550>
